src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Serialization/MessagePack/Formatters/RazorConfigurationFormatter.cs

```diff
@@ -25,6 +25,14 @@ public override RazorConfiguration Deserialize(ref MessagePackReader reader, Ser
 
         count -= 2;
 
+        var forceRuntimeCodeGeneration = false;
+
+        if (reader.NextMessagePackType is MessagePackType.Boolean)
+        {
+            forceRuntimeCodeGeneration = reader.ReadBoolean();
+            count -= 1;
+        }
```

Comment:
Why is this back again?

Reply:
Because I'm so afraid to break things!

Reply:
I think it's probably safe to remove at this point.